R/error-checks.R

```diff
@@ -138,17 +128,15 @@ isOfLength <- function(object, nbElements) {
 
 #' Check if the provided path has required extension
 #'
-#' @param file file or path name to be checked
-#' @param extension extension of the file required after "."
+#' @param file A single character of file path.
```

Comment:
A single character? Don't get it.

Reply:
In R, character is the same as string. There are no strings in R, so I am trying to be idiomatic.

For example, [xfun::file_ext()](https://cran.r-project.org/web/packages/xfun/xfun.pdf) is a popular choice to do what we are doing in this custom function, and this is what its documentation reads:

![image](https://user-images.githubusercontent.com/11330453/153875312-41df4321-85e1-407a-a70d-56b219ff68f9.png)

Should I just say `A filename`, and leave the type information out of it?

Reply:
Lol maybe. Filename is not the same a file path (with and without extension) and potentially not path. So I would make sure we make that clear with the variable name